### DISCUSSION TITLE
Oventrop wax actuators

### DISCUSSION THREAD

**Discussion by RuslanKADR** [ScratMan/HASmartThermostat#244] [General]
Hello everyone, I want to use this component with Oventrop actuators via a solid state relay. It is designed with a wax filler and heats up in 2 minutes to fully open the valve, will this thermostat be suitable and will I be able to configure the system?

**Comment by Chupaka** [comment]
Hi. Looks like what you're looking for is `pwm` (for on/off mode) and `min_cycle_duration` (so that the thermostat don't switch on/off too frequently) params.
